resource-managers/kubernetes/core/pom.xml

```diff
@@ -29,7 +29,7 @@
   <name>Spark Project Kubernetes</name>
   <properties>
     <sbt.project.name>kubernetes</sbt.project.name>
-    <kubernetes.client.version>3.0.0</kubernetes.client.version>
+    <kubernetes.client.version>4.0.0</kubernetes.client.version>
```

Comment:
4.1.0?

Reply:
hmm, agreed. Maybe 4.1.0 is a more stable version

Reply:
Yeah just wondering I don't know. What matters is what each version supports. Still no version supports 1.11 https://github.com/fabric8io/kubernetes-client/issues/1235